src/main/java/com/paneedah/mwc/bases/OreBase.java

```diff
@@ -0,0 +1,73 @@
+package com.paneedah.mwc.bases;
+
+import com.paneedah.mwc.ModernWarfareMod;
+import com.paneedah.mwc.utils.ModReference;
+import net.minecraft.block.BlockOre;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.Item;
+
+import java.util.Random;
+
+public class OreBase extends BlockOre {
+    Item itemBlock, drop;
+    int harvestLevel = 2, minDrop = 1, maxDrop = 1;
+    boolean smelt = true;
+    public OreBase(String registryName){
```

Comment:
Missing space between method and variable

Reply:
✅